src/js/components/Video/Video.js

```diff
@@ -278,13 +285,21 @@ const Video = forwardRef(
         Volume: theme.video.icons.volume,
       };
 
-      const captionControls = captions.map((caption) => ({
+      const captionControls = captions.map((caption, index) => ({
         icon: caption.label ? undefined : (
           <Icons.ClosedCaption color={iconColor} />
         ),
         label: caption.label,
         active: caption.active,
-        onClick: () => showCaptions(caption.active ? -1 : 0),
+        a11yTitle: caption.label || 'video.captions',
+        onClick: () => {
+          showCaptions(caption.active ? -1 : index);
+          // set the currently selected captions to active
+          captions[index].active = !captions[index].active;
```

Comment:
Is there a reason we aren't using the `setCaptions` function here?

Reply:
Updated this section to use `setCaptions` in latest commit